modules/react/action-bar/lib/ActionBar.tsx

```diff
@@ -1,76 +1,55 @@
 import * as React from 'react';
-import {styled} from '@workday/canvas-kit-react/common';
-import {colors, commonColors, space, CSSProperties} from '@workday/canvas-kit-react/tokens';
+import {Globals} from 'csstype';
+import {createComponent, styled, StyledType} from '@workday/canvas-kit-react/common';
+import {commonColors, colors, space} from '@workday/canvas-kit-react/tokens';
+import {HStack, HStackProps} from '@workday/canvas-kit-labs-react';
 
-export interface ActionBarProps extends React.HTMLAttributes<HTMLDivElement> {
+type PropertyPosition =
+  | Globals
+  | '-webkit-sticky'
+  | 'absolute'
+  | 'fixed'
+  | 'relative'
+  | 'static'
+  | 'sticky';
+
+interface ActionBarProps {
```

Comment:
This interface isn't accurate since all `HStack` props are valid. The only thing this interface does is add the `default` value to `position`. Should we get rid of the interface? Or maybe change it instead?

```suggestion
interface ActionBarProps extends HStackProps {
```